resources.Groups.department.ReferenceExample.ps1

```diff
@@ -1,20 +1,124 @@
-# The resourceData used in this default script uses resources based on Title
+#####################################################
+# HelloID-Conn-Prov-Target-ActiveDirectory-Resources-Groups-Department
+#
+# Version: 1.1.0
+#####################################################
 $rRef = $resourceContext | ConvertFrom-Json
 $success = $false # Set to false at start, at the end, only when no error occurs it is set to true
 
 $auditLogs = [Collections.Generic.List[PSCustomObject]]::new()
 
 # Troubleshooting
 # $dryRun = $false
-$debug = $true
+$debug = $false # Warning! Only set to true when troubleshooting, this will severly impact the performance.
 
-$path = "OU=Groups,OU=Resources,DC=consultancytest,DC=nl"
-$adGroupNamePrefix = ""
-$adGroupNameSuffix = ""
-$adGroupDescriptionPrefix = "Security Group voor combinatie "
-$adGroupDescriptionSuffix = ""
+# Variables to define what groups to query (to check if group already exists)
+$adGroupsSearchOUs = @() # Warning! When no searchOUs are specified. Groups from all ous will be retrieved.
```

Comment:
Suggestion: Add an example comment on how to use it, for example 
$adGroupsSearchOUs = @("distinguishedName1","distinguishedName2")
or
$adGroupsSearchOUs = @("OU=Groups,OU=Resources,DC=enyoi,DC=org",""OU=Groups2,OU=Resources,DC=enyoi,DC=org")

I guess I prefer the first example.